File: FITCC.UI/admin_fitcc/lib/models/kategorija.dart

import 'package:admin_fitcc/models/takmicenje.dart';
import 'package:json_annotation/json_annotation.dart';

part 'kategorija.g.dart';

@JsonSerializable(explicitToJson: true)
class Kategorija {
 int kategorijaId;
   String naziv;
   String opis;
   Takmicenje takmicenje;
  Kategorija(  this.kategorijaId, this.naziv,this.opis, this.takmicenje);

  factory Kategorija.fromJson(Map<String, dynamic> json) => _$KategorijaFromJson(json);
  Map<String, dynamic> toJson() => _$KategorijaToJson(this);
}

File: FITCCRS2App.Models/Models/Kategorija.cs
using System;
namespace FITCCRS2App.Models.Models
{
	public class Kategorija
	{
        public int KategorijaId { get; set; }
        public string? Naziv { get; set; }
        public string? Opis { get; set; }
        public int? TakmicenjeId { get; set; }

        public virtual Takmicenje? Takmicenje { get; set; }
    }
}

